internal/service/imagebuilder/distribution_configuration.go

```diff
@@ -152,6 +152,25 @@ func ResourceDistributionConfiguration() *schema.Resource {
 								},
 							},
 						},
+						"launch_template_configuration": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							MaxItems: 100,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"default": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										Default:  true,
```

Comment:
Undocumented default value from the API. I could not use `Computed: true` as it is nested inside a set and would lead to permanent diff.